sr_ur_robot_hw/src/sr_ur_driver.cpp

```diff
@@ -78,3 +78,47 @@ void UrRobotDriver::send_teach_mode_command(bool teach_mode)
 {
   ctrl_server_->send_teach_mode_command((int32_t)teach_mode);
 }
+
+void UrRobotDriver::send_payload_command()
+{
+  ctrl_server_->send_payload_command();
+}
+
+void UrRobotDriver::send_speed_command()
+{
+  ctrl_server_->send_speed_command();
+}
+
+bool UrRobotDriver::set_payload(float mass_kg, std::vector<float> center_of_mass_m)
+{
+  if (mass_kg < 0.0 || mass_kg > 8.0)
```

Comment:
Why do we choose 8.0? A UR10 is rated for 10 Kg payload isn't it?

Reply:
I made a mistake, and assumed service clients would set the payload not including the Hand, so took off ~2kg to allow some leeway. As you point out, the payload limit includes the hand, so the only sensible value is 10.0. I fixed this and moved the value limits to class constants in fb5d94cbc91ba7eef313564d6804d5.